pyphare/pyphare/pharesee/hierarchy.py

```diff
@@ -0,0 +1,162 @@
+from ..core import box as boxm
+import numpy as np
+
+class PatchData:
+    """
+    base class for FieldData and ParticleData
+    this class just factors common geometrical properties
+    """
+    def __init__(self, layout, quantity):
+        """
+        :param layout: a GridLayout representing the domain on which the data
+             is defined
+        :param quantity: ['field', 'particle']
```

Comment:
are these actually "quantities", or abstractions on the "type of quantities"?

Reply:
you mean we reserve "quantity" for Ex, Ey, etc. and field/particle are something different hence a different name should be used?

how about data_type ?